crypto_commons/generic.py

```diff
@@ -1,22 +1,33 @@
+from builtins import range as long_range
```

Comment:
This won't work for python 2 for large numbers (range is limited by maxint and creates the list in memory)